faker/providers/lorem/__init__.py

```diff
@@ -81,50 +82,80 @@ def paragraphs(cls, nb=3):
         return [cls.paragraph() for _ in range(0, nb)]
 
     @classmethod
-    def text(cls, max_nb_chars=200):
+    def text(cls, min_nb_chars=5, max_nb_chars=200):
         """
         Generate a text string.
         Depending on the $maxNbChars, returns a string made of words, sentences, or paragraphs.
         :example 'Sapiente sunt omnis. Ut pariatur ad autem ducimus et. Voluptas rem voluptas sint modi dolorem amet.'
-        :param max_nb_chars Maximum number of characters the text should contain (minimum 5)
+        :param max_nb_chars Maximum number of characters the text should contain (inclusive; default=200)
+        :param min_nb_chars Minimum number of characters the text should contain (inclusive; default=0; minimum 5)
         :return string
         """
         text = []
-        if max_nb_chars < 5:
+        if max_nb_chars < min_nb_chars:
+            raise ValueError('minimum length cannot exceed maximum length')
+        if min_nb_chars < 5:
             raise ValueError('text() can only generate text of at least 5 characters')
+        text_len = cls.random_int(min=min_nb_chars, max=max_nb_chars)
+        size = 0
 
-        if max_nb_chars < 25:
+        if text_len < 25:
             # join words
             while not text:
```

Comment:
i'm not sure this parent `while` is still needed. `text` will always be truth-y on the first iteration, the inner `while` loop should be all we need, unless I'm missing something